TeamNovaSoft/qa-exposed-server
# Adding VSCode Settings, Stylelint, and CSS Rules

The goal of this pull request is to improve the development environment by providing VSCode settings for enhanced code editing and introducing Stylelint for CSS linting. Additionally, default CSS rules have been included to assist with testing the Stylelint setup.

## Changes Made

- VSCode Settings: Added `settings.json` and `extensions.json` files to enhance the development environment with support for CSS, JavaScript, and HTML.
- Stylelint Support: Integrated Stylelint for CSS linting in the project.
- Default CSS Rules: Included a reset and some default CSS rules to facilitate testing with Stylelint.

> [!IMPORTANT]
> Try out the Stylelint rules locally. This is in preparation for setting up GitHub Actions for linting in the future.

## How to Test

- Open the project in VSCode.
- Ensure that VSCode extensions are installed based on the extensions.json.
- Experiment with the included CSS rules and observe Stylelint in action.
- Refer to the documentation note for additional information on testing Stylelint locally.

## Additional Notes

Review and feedback on the VSCode settings and Stylelint setup are welcome.
Feel free to provide suggestions for additional VSCode extensions or improvements to the linting setup.
